File: exercises/beecrowd/1174/1174.ts
// https://judge.beecrowd.com/pt/problems/view/1174
import * as fs from "fs";

function main(): void {
  const input = fs
    .readFileSync("/dev/stdin", "utf8")
    .trim()
    .split("\n")
    .map(Number);

  for (let i = 0; i < 100; i++) {
    const value = input[i];
    if (value <= 10.0) {
      console.log(`A[${i}] = ${value.toFixed(1)}`);
    }
  }
}

main();

File: exercises/beecrowd/1174/1174.go
// https://judge.beecrowd.com/pt/problems/view/1174
package main

import "fmt"

func main() {
	var a float64

	for i := 0; i < 100; i++ {
		fmt.Scan(&a)
		if a <= 10.0 {
			fmt.Printf("A[%d] = %.1f\n", i, a)
		}
	}
}
